WordPress/gutenberg
# Footnotes: Fix writingMode (text orientation setting) on the front

## What?

Closes https://github.com/WordPress/gutenberg/issues/54344
Adds the missing inline styles for the writing mode (text orientation typography setting) on the front.

## Why?

The setting works in the editor but not the front.

## How?

Checks if the attribute for the writingMode block support is set and adds the inline styles to the block wrapper.

## Testing Instructions
Activate a block theme.
The writing mode setting is disabled by default; set it to true in theme.json settings > typography > writingMode:

```
	"settings": {
		"typography": {
			"writingMode": true
		}
	}
```

Create a new post or page, and add a footnote.
Select the footnote and enable the vertical text orientation option in the typography panel.
Save and view the front.
Confirm that the text is now vertical on the front.

## Screenshots or screencast 
In the video, I am setting the text orientation setting to vertical:
https://github.com/WordPress/gutenberg/assets/7422055/e8a514bf-bf4a-44af-bcd1-6ebfb4502779
